packages/plugins/users-permissions/admin/src/components/BoundRoute/index.js

```diff
@@ -18,17 +18,17 @@ function BoundRoute({ route }) {
 
   return (
     <Stack size={2}>
-      <H3>
+      <Typography variant="delta">
         {formatMessage({
           id: 'users-permissions.BoundRoute.title',
           defaultMessage: 'Bound route to',
         })}
         &nbsp;
         <span>{controller}</span>
-        <Text style={{ fontSize: 'inherit', fontWeight: 'inherit' }} textColor="primary600">
+        <Typography variant="delta" textColor="primary600">
```

Comment:
Text to Typography variant="delta" ? 🤔 

Shouldn't it be a simple <Typography /> ?

Reply:
It's for the little .delete that you can see there, the style was set with style={{ fontSize: 'inherit', fontWeight: 'inherit' }} before to ensure it has the same fontSize/fontWeight as "Bound route to" but I thought it would make more sens to use the same Typography variant? Without it it's not the same style for both text - let me know if I doesn't make sense 🤓

<img width="432" alt="Screenshot 2021-11-15 at 15 32 04" src="https://user-images.githubusercontent.com/71838159/141799323-a5fd9f83-4bc3-4b01-b2d9-5f927ffd4af2.png">